### ISSUE TITLE
write scrapers for some other websites

### ISSUE THREAD

**Issue by moghya** [moghya/allitebooks#4]
**Following websites can be scraped**
1. http://bookboon.com

**Comment by cLupus** [comment]
Such as?

**Comment by moghya** [comment]
@cLupus Thanks for showing interest in this project.  I hope you visited http://moghya.me/allitebooks and got what we're trying to do here.

You can go through http://bookboon.com and try to wrtite scraper for it.

I'll add many such websites soon. Let me know if you gonna do it.
I'll assign this to you :)

**Comment by cLupus** [comment]
I got to take a look, on the site, as well as in your repo.
Am I correct in understanding that this issue is concerned with creating a scraper that creates a file similar to `data.py`?

**Comment by moghya** [comment]
Yes, you're correct. It's just we dump the dictionary in JSON and process that JSON.

**Comment by cLupus** [comment]
That does sound interesting. I assume the description should be in english. However, the site does offer some additional languiages, although not all the descriptions have been translated into the different languages. Is there any plan for localization (or at the very least to grab what's there in different languages)?

**Comment by moghya** [comment]
Honestly I didn't think of it. But as you have rightly raised we have to think about it ? 
What do you propose ?

**Comment by cLupus** [comment]
On closer inspection, it seems that only the site have been translated, and not the titles, or the descriptions, and such it would seem not to add much value (in the first run, anyway).

**Comment by moghya** [comment]
let's work it for English and we'll come up with solution in near future

**Comment by cLupus** [comment]
Another issue, is that http://bookboon.com 'locks' their books behind a dropdown, and do not offer direct links to their books.
There are some ways to aliviate it
1. Download the zip-files, and host them (somewhere) by link.
2. Do some trickery with cookies that are sent along with the request
3. Something else?

**Comment by moghya** [comment]
downloading zip one option but, maybe intercepting the request which downloads the book will solve our problem. 
Think it this way:
scraper won't follow what bookboon, it'll work a step ahead we can workaround and get to know what exactly happens after filling the details and instead of filling the details we can directly send the request to download pdf.

**Comment by EmilLuta** [comment]
Hi there, ladies and gentlemen. What's the status on this issue? @moghya Mind if I hop in? Also, shouldn't the first page be a bit more descriptive? I.E. A huge majority of web pages should have written somewhere in the homepage what it is and what it does, not down in the code.

Let me know what you think!

**Comment by moghya** [comment]
@EmilLuta maybe you can contribute by working on #3.

**Comment by sandeep661** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://link.bnbreward.online/0mhPJ2)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://link.bnbreward.online/0mhPJ2) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @BFF271, @NotII, @misterhuhuhu, @AndreLaranjeira, @ozgur-yalcin, @dbaspider, @nrgapple, @xautofzx, @brossetti1, @democloudx, @jiahut, @raashidanwar, @SkipM4, @sai-prasanna, @thomas-gale, @matcarfer, @mxpuck, @HighGuy710, @WyattBlue, @blueyed, @padulkemid, @lapinpt, @vanohj, @imknown, @falcolee, @bachhuberdesign, @thmsl, @MekkyUA, @jopijuco, @oneironautd, @shaobin0604, @AlexJarrah, @zhangmx, @naveen521kk, @burakozkirdeniz, @mar109us, @BIOLorenz, @Parham-Bakhshaei, @CrossingTrees, @WPTP, @cubical8569, @Nangelov7, @farwyler, @roushjosh, @amohammad, @yasincidem, @TruckingOtter, @meld-cp, @worldtok, @Khoailan
